fiftyone/utils/transformers.py

```
@@ -1489,13 +1488,36 @@ def __call__(
+                ):
+                    batched_labels = [text_labels] * len(image_sizes)
+                else:
+                    batched_labels = text_labels
+
+                output = self._objection_detection_processor(
+                    output,
+                    text_labels=batched_labels,
+                    threshold=confidence_thresh or 0,
+                    target_sizes=image_sizes,
+                )
+            except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)